examples/preview/sass/_font-face.scss

```diff
@@ -0,0 +1,143 @@
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans ExtraLight Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans ExtraLight Italic'), url('/fonts/public-sans-v1.008/PublicSans-ExtraLightItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Light';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Light'), url('/fonts/public-sans-v1.008/PublicSans-Light.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Light Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Light Italic'), url('/fonts/public-sans-v1.008/PublicSans-LightItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Medium';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Medium'), url('/fonts/public-sans-v1.008/PublicSans-Medium.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Medium Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Medium Italic'), url('/fonts/public-sans-v1.008/PublicSans-MediumItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans SemiBold';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans SemiBold'), url('/fonts/public-sans-v1.008/PublicSans-SemiBold.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans SemiBold Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans SemiBold Italic'), url('/fonts/public-sans-v1.008/PublicSans-SemiBoldItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Bold';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Bold'), url('/fonts/public-sans-v1.008/PublicSans-Bold.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Bold Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Bold Italic'), url('/fonts/public-sans-v1.008/PublicSans-BoldItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans ExtraBold';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans ExtraBold'), url('/fonts/public-sans-v1.008/PublicSans-ExtraBold.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans ExtraBold Italic';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans ExtraBold Italic'), url('/fonts/public-sans-v1.008/PublicSans-ExtraBoldItalic.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
+	font-family: 'Public Sans Black';
+	font-style: normal;
+	font-weight: normal;
+	src: local('Public Sans Black'), url('/fonts/public-sans-v1.008/PublicSans-Black.woff') format('woff');
+}
+
+@font-face {
+	font-display: swap;
```

Comment:
Couple things on the font:
1) Do we need _all_ the different weights? I typically only include the ones needed like regular, bold and italic.
2) In an effort to keep the amount of files this starter theme adds to a minimum, what do you think of just adding a `link` tag in the `head` to Google's hosted fonts? https://fonts.google.com/specimen/Public+Sans

Reply:
Good points, thanks.

I'm good with linking to Google Fonts instead. It looks like we'll need to amend `WPHead` to pass children to `Head` before I can do that, though (we don't have a good way to add extra elements to `<head>` at the moment). I'll spin up a quick PR to do that.

Reply:
Swapped local fonts for Google Fonts in https://github.com/wpengine/headless-framework/pull/109/commits/af85a5d0b5c439f49ff91be8de740aa4bb7d914e.